src/commands/env/list.js

```diff
@@ -22,9 +22,17 @@ class EnvListCommand extends Command {
     })
 
     const siteData = await api.getSite({ siteId })
-    const {
-      build: { environment = {} },
-    } = config
+    const environment = Object.entries(cachedConfig.env).reduce((env, [key, variable]) => {
+      // Omitting general variables to reduce noise.
+      if (variable.sources[0] === 'general') {
+        return env
+      }
+
+      return {
+        ...env,
+        [key]: variable.value,
+      }
+    }, {})
```

Comment:
[sand] Could we use `const fromEntries = require('@ungap/from-entries')` for this?

```suggestion
    const environment = fromEntries(
      Object.entries(cachedConfig.env)
        .filter(([, variable]) => variable.sources[0] !== 'general')
        .map(([key, variable]) => [key, variable.value]),
    )
```

Reply:
Sure! I'll update it.